pages/add/form-fields/detail-info-fields.js

```diff
@@ -0,0 +1,108 @@
+module.exports = {
+  creators: {
+    type: `text`,
+    label: `Who are the creators? This could be staff, contributors, partners…`,
+    placeholder: `Name`,
+    fieldClassname: `form-control`,
+    multiplicity: 1,
+    addLabel: `+ Add another`,
+    // optional: true
+  },
+  interest: {
+    type: `text`,
+    label: `Why might this be interesting to other people in our network?`,
+    placeholder: ``,
+    fieldClassname: `form-control`,
+    validator: {
+      error: `Maximum 300 characters.`,
+      validate: function(value) {
+        if (!value) {
+          // this is an optional field, it can be left blank
+          return;
+        }
+
+        return value && value.length > 300; // return true if value fails validation
+      }
+    },
+  },
+  tags: {
+    type: `text`,
+    label: `Keywords to help with search by program, event, campaign, subject …`,
+    placeholder: `#mozfest  #code  #tool`,
+    fieldClassname: `form-control`
+  },
+  issues: {
+    type: `checkboxGroup`,
+    label: `Check any Key Internet Issues that relate to your project.`,
+    options: [ `Online Privacy & Security`, `Open Innovation`, `Decentralization`, `Web Literacy`, `Digital Inclusion` ],
+    colCount: 1,
+    fieldClassname: `form-control`,
+    // optional: true
+  },
+  'get_involved': {
+    type: `text`,
+    label: `Looking for support? Describe how people can do that.`,
+    placeholder: `Contribute to the code.`,
+    fieldClassname: `form-control`,
+    validator: {
+      error: `Maximum 300 characters.`,
+      validate: function(value) {
+        if (!value) {
+          // this is an optional field, it can be left blank
+          return;
+        }
+
+        return value && value.length > 300; // return true if value fails validation
+      }
+    },
+    // optional: true
+  },
+  'get_involved_url': {
+    type: `text`,
+    label: `Link for people to get involved.`,
+    placeholder: `https://example.com`,
+    fieldClassname: `form-control`,
+    validator: {
+      error: `Not a valid URL. Remember to include protocol (http:// or https://).`,
+      validate: function(value) {
+        if (!value) {
+          // this is an optional field, it can be left blank
+          return;
+        }
+
+        let r = new RegExp(/^((http||https):\/\/)([\da-z\.-]+)\.([a-z\.]{2,6})([\/\w \.-]*)*\/?$/);
```

Comment:
this makes me nervous... what are we testing for here? I'm almost certain that we can use the URL api to test for what we want to test for here.